runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -45,14 +41,17 @@
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 
+import org.corfudb.comm.ChannelImplementation;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.comm.ChannelImplementation'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)